src/patternfly/components/Page/page.scss

```diff
@@ -287,6 +295,39 @@
     }
   }
 
+  .pf-c-button {
+    min-width: var(--pf-c-page__header-tools--c-button--MinWidth);
+    padding-right: var(--pf-c-page__header-tools--c-button--PaddingRight);
+    padding-left: var(--pf-c-page__header-tools--c-button--PaddingLeft);
+
+    &.pf-m-selected {
+      &::before {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        content: "";
+        background-color: var(--pf-c-page__header-tools--c-button--m-selected--before--BackgroundColor);
+        border-radius: var(--pf-c-page__header-tools--c-button--m-selected--before--BorderRadius);
+      }
+
+      /* stylelint-disable */
```

Comment:
Can you check in this file and convert any `/* comment */` line to `// comment`? That'll make it a sass comment which won't get compiled into patternfly.css